indico/types/dataset.py

```diff
@@ -115,20 +124,35 @@ class ReadApiOcrOptionsInput(BaseType):
         upscale_images(bool): Scale up low resolution images.
         languages(List[str]): List of languages to use.
     """
+
     auto_rotate: bool
     single_column: bool
     upscale_images: bool
     languages: List[str]
 
+
 class OcrInputLanguage(BaseType):
     name: str
     code: str
 
-class OcrOptionsInput():
+
+class IncludeSections(BaseType):
+    header: Optional[bool]
+    body: Optional[bool]
+    attachments: Optional[bool]
+
+
+class EmailOptions(BaseType):
```

Comment:
This would benefit with an Args comment. Unpack is a bit ambiguous, so it would be good to have a reminder later what the heck we were thinking about.